internal/provider/harness_k3s_resource.go

```diff
@@ -60,6 +60,7 @@ type HarnessK3sResourceModel struct {
 	Timeouts             timeouts.Value                           `tfsdk:"timeouts"`
 	Resources            *ContainerResources                      `tfsdk:"resources"`
 	Hooks                *HarnessHooksModel                       `tfsdk:"hooks"`
+	KubeletConfig        types.String                             `tfsdk:"kubelet_config"`
```

Comment:
this is fine staying as a string that we probably validate can be marshal'd. I don't know of an easy way to `types.Any` or `types.Json` in tf